lms/djangoapps/support/static/support/jsx/entitlements/components/Search/Search.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { InputText } from '@edx/paragon';
+
+
+class Search extends React.Component{
+	constructor(props) {
+		super(props)
+		this.state = { username: '' }
+	}
+
+	handleSubmit(event) {
+		event.preventDefault(); //prevents the page from refreshing, refresh unneeded since state update will trigger react to update dom
```

Comment:
Line length seems to be pushing longer than 120.  Please consider shortening or moving above rather than inline.